clients/socket-client.rst

```diff
@@ -1,9 +1,12 @@
-Socket Client
-=============
+Socket Client (deprecated)
+==========================
 
 The socket client uses the stream extension from PHP, which is integrated into
 the core.
 
+This client only implements the PHP-HTTP synchronous interface, which has been
+superseded by PSR-18. Use one of the PSR-18 clients instead.
```

Comment:
@joelwurtz @Nyholm i think you 2 mostly worked on the socket client. as it does not implement async and is quite inactive, i think we should deprecate it in favor of using PSR-18 clients.

Reply:
the package should be updated, or marked abondonded.  Not sure what we're waiting for :shrug:

Reply:
i wanted to give some time to react for the authors of the package who are part of the php-http group.

but no reaction is also a kind of statement on priorities. i can't promise that i can get to it very soon however, as i have a holidays coming up.

Reply:
https://github.com/php-http/socket-client/pull/70